xclim/indices/_multivariate.py

```diff
@@ -5,6 +5,7 @@
 
 from xclim import run_length as rl
 from xclim import utils
+from xclim.utils import _rolling
```

Comment:
This contradicts #334! I guess a merge of master will erase it.

Reply:
Yep. I need to re-merge master and do some adjustments before this goes live. You'll need to change your Back preferences in all future PRs once this is merged.